goodmap/db.py

```
@@ -104,6 +102,404 @@ def get_locations(db, location_model):
+        raise ValueError(f"Location with uuid {uuid} not found")
+
+    map_data[idx] = location.model_dump()
+    json_file["map"]["data"] = map_data
+
+    with open(self.data_file_path, "w") as file:
+        json.dump(json_file, file)
+
+
+def json_db_update_location(self, uuid, location_data, location_model):
+    location = location_model.model_validate(location_data)
+    idx = next((i for i, point in enumerate(self.data.get("data", [])) if point.get("uuid") == uuid), None)
```

Ruff (0.11.9)

Line too long (107 > 100)

(E501)